src/main/java/com/mitchej123/hodgepodge/mixins/early/minecraft/MixinWorld_FixXray.java

```diff
@@ -0,0 +1,32 @@
+package com.mitchej123.hodgepodge.mixins.early.minecraft;
+
+import net.minecraft.block.Block;
+import net.minecraft.util.AxisAlignedBB;
+import net.minecraft.world.World;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Unique;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+import com.mitchej123.hodgepodge.common.BlockInvoker;
+
+@Mixin(World.class)
+public class MixinWorld_FixXray {
+
+    @Unique
+    private static final AxisAlignedBB hodgepodge$dummyAABB = AxisAlignedBB.getBoundingBox(0, 0, 0, 0, 0, 0);
```

Comment:
Nit: In Java, constants are all uppercase by convention: `hodgepodge$DUMMY_AABB`